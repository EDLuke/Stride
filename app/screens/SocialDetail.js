import React from 'react';
import {Font} from 'expo';
import { Button, ScrollView, StyleSheet, Text, View } from 'react-native';
import { NavigationActions } from 'react-navigation';

export default class SocialDetail extends React.Component{
	


	render() { 
		return (
		<View style = {styles.container}>
			    <View style = {styles.titleContainer}>
	          <Text style = {styles.boxtext}>Loading</Text>
	        </View>
	        <View style = {styles.loginContainer}>
	          <Button style={styles.loginButton}
              	onPress={() => this.props.navigation.dispatch(NavigationActions.back())}
              	title="Return"
            	/>
	        </View>
	    </View>
		);
	}
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
    alignItems: 'flex-start',
    justifyContent: 'flex-start',
    marginTop: 64
  },
  title: {
    backgroundColor: 'rgba(0,0,0,0)',
    fontFamily: 'Helvetica',
    fontSize: 50,
    fontWeight: '900',
    color: '#fff'
  },
  boxtext:{
    fontFamily: 'Helvetica',
    fontSize: 20,
    color: '#686868'
  },
  subtitle: {
    fontFamily: 'Helvetica',
    fontSize: 20,
    color: '#fff'
  },

  loginContainer:{
    backgroundColor: 'rgba(0,0,0,0)',
    marginTop: 15,
  },
  signupContainer:{
    backgroundColor: 'rgba(0,0,0,0)',
    marginTop: 35,
  },
  titleContainer:{
    backgroundColor: 'rgba(0,0,0,0)',
    width: 250,
    marginTop: 35
  },
  signupContainer:{
    backgroundColor: 'rgba(0,0,0,0)',
    width: 250,
    marginTop: 35
  },
  emailContainer: {
    backgroundColor: 'rgba(0,0,0,0)',
    height: 30,
    width: 250,
    marginTop: 50,
    borderBottomWidth: 1,
    borderColor: "#FFF",
  },
  passwordContainer: {
    backgroundColor: 'rgba(0,0,0,0)',
    height: 30,
    width: 250,
    marginTop: 35,
    borderBottomWidth: 1,
    borderColor: "#FFF",
  },
  subtitleContainer:{
    backgroundColor: 'rgba(0,0,0,0)',
    marginTop: 15,
  },
  loginButton:{
    borderRadius:0,
  },
  signupText:{
    textAlign: 'center',
    color: '#FFF',
    opacity: 0.5,
  },

});