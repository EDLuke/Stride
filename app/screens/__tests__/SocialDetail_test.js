import 'react-native';
import React from 'react';
import { shallow } from 'enzyme';
import SocialDetail from '../SocialDetail';
import { View } from 'react-native';

import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const wrapper = shallow(
      <SocialDetail/>
    );
  expect(wrapper).toMatchSnapshot();
});