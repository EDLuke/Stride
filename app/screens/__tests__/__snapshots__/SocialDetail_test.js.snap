// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "flex-start",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "flex-start",
      "marginTop": 64,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0)",
        "marginTop": 35,
        "width": 250,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#686868",
          "fontFamily": "Helvetica",
          "fontSize": 20,
        }
      }
    >
      Loading
    </Text>
  </View>
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0)",
        "marginTop": 15,
      }
    }
  >
    <Button
      onPress={[Function]}
      style={
        Object {
          "borderRadius": 0,
        }
      }
      title="Return"
    />
  </View>
</View>
`;
